addons/stock_account/__init__.py

```diff
@@ -39,3 +39,30 @@ def _configure_journals(env):
             data['account.journal'] = ChartTemplate._get_stock_account_journal(template_code)
         ChartTemplate._load_data(data)
         ChartTemplate._post_load_data(template_code, company, template_data)
+
+    # if we already have quantity on hand for storable product, create an initial valuation layer
+    valued_locations = env["stock.location"].search([]).filtered(lambda loc: loc._should_be_valued())
+    qty_by_product = env["stock.quant"]._read_group(
+        [
+            ("product_id.type", "=", "product"),
```

Comment:
product_id.type is no longer used to know if the product is storable or not, you can use the `is_storable` field instead.

```suggestion
            ("product_id.is_storable", "=", True),
```